blocks/top-table-list-block/themes/news.json

```diff
@@ -366,15 +387,18 @@
 			"default": {
 				"components": {
 					"heading": {
-						"align-self": "left",
-						"justify-content": "left"
+						"align-self": "inline-start",
+						"justify-content": "inline-start"
```

Comment:
The values for these are just `start`. What's strange is that `left` is not a valid value for either of these properties, so this should have never worked.

Reply:
@vgalatro Updated [4bf7fbb](https://github.com/WPMedia/arc-themes-blocks/pull/1830/commits/4bf7fbb991a802ef23fdbcf606d6de0e075127e5)